tests/test_components/test_source.py

```diff
@@ -151,3 +151,81 @@ def get_pol_dir(axis, pol_angle=0, angle_theta=0, angle_phi=0):
     assert np.allclose(
         get_pol_dir(axis=2, angle_theta=np.pi / 4), (-1 / np.sqrt(2), 0, +1 / np.sqrt(2))
     )
+
+
+def test_BroadbandSource():
```

Comment:
Sorry, very nitpicky, but the style we follow in python is `snake_case` for functions and `CamelCase` for class names only, so this would be `test_broadband_source`